modules/rosa-sdpolicy-am-aws-compute-types.adoc

```diff
@@ -251,7 +255,13 @@
 - r7iz.16xlarge (64 vCPU, 512 GiB)	 
 - r7iz.32xlarge (128 vCPU, 1024 GiB)	 
 - r7iz.metal-16xl (64 vCPU, 512 GiB)
-- r7iz.metal-32xl (128 vCPU, 1024 GiB)
+- r7iz.metal-32xl (128 vCPU, 1,024 GiB)
+- z1d.metal (48&#135; vCPU, 384 GiB)
+- z1d.xlarge (4 vCPU, 32 GiB)
+- z1d.2xlarge (8 vCPU, 64 GiB)
+- z1d.3xlarge (12 vCPU, 96 GiB)
+- z1d.6xlarge (24 vCPU, 192 GiB)
+- z1d.12xlarge (48 vCPU, 384 GiB)
 
 &#8224; These instance types provide 96 logical processors on 48 physical cores. They run on single servers with two physical Intel sockets.
```

Comment:
You changed the language in the general section from provide to offer. You may want to match throughout.